web/src/lib/domain-logic/routes.ts

```diff
@@ -7,6 +7,7 @@ export const ROUTES = {
   dashboard: "/dashboard",
   onboarding: "/onboarding",
   unsupported: "/unsupported",
+  quickActions: "/actions",
```

Comment:
I think this should also be added to findDeadLinks.ts

Reply:
good catch!